### DISCUSSION TITLE
Model Results incompatible with dfs2

### DISCUSSION THREAD

**Discussion by daniel-caichac-DHI** [DHI/modelskill#123] [Ideas]
Hi all,

I wanted to connect track observations (csv) to a model result in `dfs2` format instead of `dfsu`, and it does not work :(
I know that this can be done outside fmskill but it would be nice that the `ModelResult` class could handle `dfs2` files as well (I imagine that it could be even simpler than comparing against a dfsu, or at least not any more complex, since it is gridded data).

**Comment by simj-dhi** [comment]
I have also run into the issue of not having dfs2's supported in the ModelResult class. Would very much like to see this added :)

**Reply by jsmariegaard** [reply]
Agree - it would be great - and thanks for letting me know that you need it. I will prioritize it higher then 😸

**Comment by ecomodeller** [comment]
https://github.com/DHI/modelskill/pull/428
